applications/nrf_desktop/README.rst

```diff
@@ -339,7 +339,9 @@ The project comes with configuration files for the following boards:
     * PCA20037 - nRF Desktop keyboard reference design
     * PCA20044 - nRF Desktop casual mouse reference design (nRF52832)
     * PCA20045 - nRF Desktop casual mouse reference design (nRF52810)
-    * PCA10059 - nRF Desktop dongle reference design
+    * PCA10059 - nRF Desktop dongle reference design (nRF52840)
+    * PCA10111 - nRF Desktop dongle reference design (nRF52833)
```

Comment:
What is this?

Reply:
PCA10111 is a board created for our project. It looks as PCA10059 but has nrf52833 on board.
The application works as HID dongle here.